src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/DescriptorsValidation.feature.disabled

```diff
@@ -0,0 +1,285 @@
+                      {
+                          "namespace": "uri://ed-fi.org/DisabilityDescriptor",
+                          "codeValue": "Deaf-Blindness Test",
+                          "shortDescription": "Deaf-Blindness Test",
+                          "description": "Deaf-Blindness Test"
+                      }
+                  """
+             Then it should respond with 204
+
+        @ignore
+        Scenario: 04 / Ensure clients can create a descriptor using optional attributes
+             When a POST request is made to "/ed-fi/disabilityDescriptors" with
+                  """
+                  {
+                      "namespace": "uri://ed-fi.org/DisabilityDescriptor",
+                      "codeValue": "Testing Deaf-Blindness",
+                      "shortDescription": "Testing Deaf-Blindness",
+                      "description": "Testing Deaf-Blindness",
+                      "effectiveBeginDate": "2023-07-24",
+                      "effectiveEndDate" "2030-07-24"
+                  }
+                  """
+             Then it should respond with 201
+
+        @ignore
+        Scenario: 05 / Ensure clients can retrieve optional attributes from descriptors that have used them
+            Given the system has this codeValue "Testing Deaf-Blindness"
+             When a GET request is made to "/ed-fi/disabilityDescriptors/{id}"
+             Then it should respond with 200
+              And the response body is
+                  """
+                  {
+                      "namespace": "uri://ed-fi.org/DisabilityDescriptor",
+                      "codeValue": "Testing Deaf-Blindness",
+                      "shortDescription": "Testing Deaf-Blindness",
+                      "description": "Testing Deaf-Blindness",
+                      "effectiveBeginDate": "2023-07-24",
+                      "effectiveEndDate" "2030-07-24"
+                  }
+                  """
+
+        @ignore
+        Scenario: 06 / Ensure clients can add optional attributes to descriptors that do not include them
+            Given the system has this codeValue "Deaf-Blindness"
+             When a PUT request is made to "/ed-fi/disabilityDescriptors/{id}" with
+                  """
+                  {
+                      "namespace": "uri://ed-fi.org/DisabilityDescriptor",
+                      "codeValue": "Deaf-Blindness",
+                      "shortDescription": "Deaf-Blindness",
+                      "description": "Deaf-Blindness"
+                      "effectiveBeginDate": "2024-07-24",
+                      "effectiveEndDate" "2025-12-31"
+                  }
+                  """
+             Then it should respond with 204
+
+        @ignore
+        Scenario: 07 / Ensure clients can retrieve a descriptor by requesting through a valid codeValue
+            Given the system has this codeValue "Deaf-Blindness"
+             When a GET request is made to "/ed-fi/disabilityDescriptors/{id}"
```

Comment:
Wrong URL style. You have the right style on line 132.

Reply:
Changed